schemas/glean/glean/glean.1.schema.json

```diff
@@ -752,12 +752,19 @@
                 "type": "string"
               },
               "extra": {
-                "properties": {
-                  "type": {
-                    "type": "string"
+                "anyOf": [
```

Comment:
I don't know if the jsonschema-transpiler supports anyOf. In testing locally, it looks like this ended up turning `extra` into a STRING field rather than a RECORD. @acmiyaguchi should be able to comment on whether we have a good way of expressing an object that can be null.

Reply:
It looks like `oneOf` is supported and produces identical BQ schema output. Does `oneOf` make sense in this situation?

Reply:
`"type": ["null", "object"]` is also a valid construct that produces the same schema. `"oneOf": [...]` would also make sense here, as these two fragments are treated equivalently.

https://gist.github.com/acmiyaguchi/a273dfbd8438769f6047566021cfa72d

Reply:
`oneOf` works for me.  I'll update this.